config/jobs/kubernetes-sigs/ingress-controller-conformance/ingress-controller-conformance.yaml

```diff
@@ -59,7 +59,7 @@ presubmits:
         args:
         - bash
         - -c
-        - apt update && apt-get install -y python3-pip && pip3 install reformat-gherkin && make verify-gherkin
+        - apt update && apt-get install -y python3-pip && pip3 install reformat-gherkin==2.0.0 --ignore-installed PyYAML && make verify-gherkin
```

Comment:
@aledbf I think we probably broke this [here](https://github.com/kubernetes/test-infra/pull/18604) :/ I _think_ you can skip installing `python3-pip` here though now as it should be present in the image. However, it [looks like](https://prow.k8s.io/view/gcs/kubernetes-jenkins/pr-logs/directory/pull-ingress-controller-conformance-gherkin/1292184216368320512#1:build-log.txt%3A32) installing it again will just be a no op so probably fine to keep it around :)

Reply:
done :)

Reply:
@hasheddan can you approve this change?

Reply:
Sure! Although I'm not in approver here so you'll need someone else as well :)